PocketKit/Sources/Textile/Style/PocketButtonStyle.swift

```diff
@@ -105,6 +114,15 @@ public struct PocketButtonStyle: ButtonStyle {
                     .foregroundColor(configuration.isPressed ? Constants.InternalInfoLink.pressedForegorundColor : Constants.InternalInfoLink.foregroundColor)
                     .cornerRadius(Constants.cornerRadius)
             }
+            .if(variation == .destructive) { view in
+                view
+                    .font(Font(Constants.style.fontDescriptor))
+                    .frame(height: Constants.buttonHeight)
+                    .tint(Constants.Destructive.tintColor)
+                    .background( configuration.isPressed ? Constants.Destructive.pressedBackgroundColor : Constants.Destructive.backgroundColor)
+                    .foregroundColor(configuration.isPressed ? Constants.Destructive.pressedForegroundColor : Constants.Destructive.foregroundColor)
+                    .cornerRadius(Constants.cornerRadius)
```

Comment:
We should add this style to the swift previews below!

Reply:
dang